docs/source/cleanlab/datalab/guide/issue_type_description.rst

```diff
@@ -38,11 +38,15 @@ Each input is optional, if you do not provide it, Datalab will skip checks for t
 Label Issue
 -----------
 
+.. jinja:: label_context
+    :file:  cleanlab/datalab/guide/_templates/issue_types_tip.rst
```

Comment:
can we move this to the end of this Label Issue section (right before Outlier Issue section begins below). That way these tips appear after the description of the issue which makes more sense to me. 

(Same idea would apply for all issue types)

Reply:
Moved all tips to the end of their respective sections.